code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -92,6 +92,11 @@
 		stance_step = 0
 
 /mob/living/simple_animal/hostile/bear/Life()
+	..()
+	if (life_tick % 30 == 0)//A point of anger wears off per minute

```

Comment:
Wonder if this even needs to be in `Life()` instead of `think()`, considering that it only really applies to the AI.